### PR TITLE
Update genomics docs

### PR DESCRIPTION
**Pull request by nfahlgren**
This pull request updates the genomics information in the data sources and experimental design pages. It reorganizes the whole-genome resequencing samples table into a single page and adds a page for a genotyping-by-sequencing samples table. Information for accessing genomics data through various services is updated.

This pull request should not conflict with the current pull requests from @phloz or @dlebauer but should be merged after.

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -32,13 +32,13 @@
...
     * [Point Cloud Data](products/point-cloud-data.md)
   * [How to Access Data](user/how-to-access-data.md)
-    * [Using Clowder (Sensor and Genoomics data)](user/using-clowder.md)
-    * [Using Globus (Sensor and Genomics data)](user/using-globus.md)
-    * [Using BETYdb (trait data, experimental metadata)](user/using-betydb.md)
-       * [Accessing BETYdb via ArcMap and other GIS software](accessing-betydb-with-arcmap.md) 
-    * [Using CoGe (Genomics)](user/using-coge.md)
-    * [Using CyVerse (Genomics)](user/using-cyverse.md)
-    * [Using Analysis Workbench (all data)](user/using-analysis-workbench.md)
+    * [Using Clowder \(Sensor and Genoomics data\)](user/using-clowder.md)
```
Comment (dlebauer) [inline SUMMARY.md]: escaping the `()` appears to have no effect when compiled, but are the `\` necessary?

Comment (nfahlgren) [reply SUMMARY.md]: Hmm, I didn't intentionally put those in. Not sure if they were already there or if GitBook put them in. Just noticed a typo there in "Genoomics"

**Comment by nfahlgren** [comment]
@dlebauer for linking to BETYdb I was going to add direct links to each cultivar in the table, but the records are not public so I did not for now. Maybe we could anyway? Users would just need to log in first.

For the cultivar.name field, not sure. The BAP are okay right? Since they just have the USDA GRIN identifiers (PI######). The cultivar names in the RIL table should be evaluated by @zhenbinHU, I just grabbed the unique identifiers from the GBS key file.

**Comment by dlebauer** [comment]
@nfahlgren only the traits and yields tables are restricted. All other tables can be queried with a key that has lots of 9s.

**Comment by dlebauer** [comment]
The public key is 9999999999999999999999999999999999999999

**Comment by nfahlgren** [comment]
@dlebauer how's this look? The first column now has direct links to the cultivar record in BETYdb for each accession.

**Comment by dlebauer** [comment]
@nfahlgren that looks good. Two notes:

Do you think it would be better to use the cultivar name in the lookup rather than hard-coding the id? e.g. https://terraref.ncsa.illinois.edu/bety/api/beta/cultivars?key=9999999999999999999999999999999999999999&limit=none&name=PI145633

would work (would be easier to maintain, more robust, but sends user to json rathr than to the 'view' html page).

Also, is there a reason that you didn't add links for the GBS data?

**Comment by nfahlgren** [comment]
@dlebauer updated the links to do the lookup. I'm not 100% sure about the GBS data. I built the table from the key file that is associated with the raw data. It looks like the RILs in this population are not in the cultivars table but am not sure if that's true or whether the line names just don't match. If @zhenbinHU can confirm that the table is correct then we can add them to BETYdb and add links to the table.

**Comment by gsrohde** [comment]
FYI, if you want HTML, https://terraref.ncsa.illinois.edu/bety/cultivars?key=9999999999999999999999999999999999999999&limit=none&search=name%3dPI145633 will take you to a one-item list of cultivars, with cultivar PI145633 being the item.  You can then click on the view icon to get additional info.  ("%3d" is the URL-encoded form of the "=" sign.)

**Comment by nfahlgren** [comment]
@gsrohde great! I updated the URLs.

**Comment by nfahlgren** [comment]
@dlebauer pretty sure they are all fixed now. The script I was using was not always working when there was more than one search result. I used a different method now by downloading data for all sorghum accessions and cross-referencing the accession names between the two tables.

**Comment by dlebauer** [comment]
Thanks @nfahlgren!

@em-bellis thanks for reporting the broken links ... the update links are here: https://terraref.gitbooks.io/terraref-documentation/content/user/sorghum-lines-genomics.html please let us know if you find any additional problems.

**Pull request merged**